CHANGELOG.md

```diff
@@ -11,6 +11,25 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🎁 The `zmq-app` and `zeek` plugins now use the Unix select system call for
+  improved performance during message passing. Both plugins forward messages a
+  lot faster to subscribers than before.
```

Comment:
It'd be nice to switch from a qualitative "a lot" statement to a quantitative one.